src/index.js

```diff
@@ -39,8 +40,12 @@ function App() {
     history.push('/login');
   };
 
+  // This state will hold the title that appears in the <Header />
+  const [headerTitle, setHeaderTitle] = useState('Story Squad');
+
   return (
     <Security {...config} onAuthRequired={authHandler}>
+      <Header title={headerTitle} />
```

Comment:
Everything looks good, only concern I have is the header showing up on the parent dash when it isn't supposed to. But that can be something everyone can look at together when getting our routing in order.

Reply:
We can look into some way to hide it on the parent dashboard. Every other page on the app has it and by moving it to `<App />`  we don't need to keep typing it out in each component.